pkg/k8s/configmap/configmap_test.go

```diff
@@ -6,8 +6,8 @@ import (
 	"reflect"
 	"testing"
 
-	mock "github.com/ethersphere/beekeeper/mocks/k8s"
 	"github.com/ethersphere/beekeeper/pkg/k8s/configmap"
+	mock "github.com/ethersphere/beekeeper/pkg/k8s/mocks"
```

Comment:
You can now remove this `mock` alias and use `mocks` directly. Do this on every place for this package.